app/controllers/admin/facility_groups_controller.rb

```diff
@@ -33,6 +38,7 @@ def create
 
   def update
     if @facility_group.update(facility_group_params) && @facility_group.toggle_diabetes_management
+      update_block_regions if Flipper.enabled?(:regions_prep)
```

Comment:
We gotta handle the case where the update fails from the update_block_regions here - probably want to wrap the whole thing in a transaction?